exchanges/hitbtc/hitbtc.go

```diff
@@ -253,18 +249,6 @@ func (h *HitBTC) GenerateNewAddress(ctx context.Context, currency string) (Depos
 	return resp, err
 }
 
-// GetActiveorders returns all your active orders
```

Comment:
I have added that you've removed these to your PR since you didn't mention it 💥🗞️